### ISSUE TITLE
Unresolved Merge?

### ISSUE THREAD

**Issue by joelim01** [learn-co-curriculum/devise_roles_lab#17]
lines 175-179
config/initializers/devise.rb
```
<<<<<<< HEAD
  config.reconfirmable = true
=======
  config.reconfirmable = false
>>>>>>> origin/solution
```

**Comment by TheLucasMoore** [comment]
@PeterBell @aturkewi Resolved in #18 

Thanks @joelim01!

**Comment by PeterBell** [comment]
Perfect - merged - thanks!
